docs/_config.yml

```
@@ -21,8 +21,8 @@
 title: SSW Dory
 description: >- # this means to ignore newlines until "baseurl:"
   SSW's intelligent workflow solution designed to streamline collaboration and reduce roadblocks in your daily operations.
-baseurl: "/docs" # the subpath of your site, e.g. /blog
-url: "https://SSWDory.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+baseurl: "/SSW.Dory/docs" # the subpath of your site, e.g. /blog
```

I believe this should just be `ssw.dory` 

Tested using : https://sswconsulting.github.io/SSW.Dory/assets/css/style.css?v=1d2836587bbc20d490f9cb46a1d917332580b9a0

This seems to be working. 

```suggestion
baseurl: "/SSW.Dory" # the subpath of your site, e.g. /blog
```